dependencies.yaml

```diff
@@ -160,6 +231,46 @@ dependencies:
           - {matrix: {cuda: "11.4"}, packages: *build_python_packages_cu11}
           - {matrix: {cuda: "11.2"}, packages: *build_python_packages_cu11}
           - {matrix: null, packages: [*cudf_conda, *rmm_conda] }
+  build_python_cuproj:
```

Comment:
Does cuproj have any dependency on cuspatial, or is it completely independent?

Reply:
Independent